Safari/Info.plist

```diff
@@ -13,9 +13,9 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleShortVersionString</key>
-	<string>7.7.3</string>
+	<string>7.7.4</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>7.7.4</string>
```

Comment:
I'm 90% sure that CFBundleVersion is meant to be 1. The way that I update this file by letting Safari's developer tools do its automatic thing so I don't actually know. I'm hosed at work right now but should be able to properly look at this on the weekend.